indigo_api/exporters.py

```diff
@@ -87,9 +87,15 @@ def render_coverpage(self, document):
     def get_context_data(self, **kwargs):
         """ Get the context data passed to the HTML template.
         """
+        # only commencements up to the current point in time should be considered for the coverpage
+        commencements = None
+        if kwargs.get('document'):
+            document = kwargs.get('document')
+            commencements = document.work.commencements.filter(date__lte=document.expression_date)
```

Comment:
This doesn't do quite what we want. Instead, I think we need a helper  method on the work which (optionally) filters commencements to only those with provisions at or before a particular date.